test: added tests for credit card form, login form, registration from

ilkovecVladislav opened the pull request.

ilkovecVladislav committed:
test: added tests for credit card form, login form, registration from

satansdeer commented on src/modules/logIn/LogIn.test.tsx:
```diff
@@ -0,0 +1,72 @@
+  it('renders correctly', () => {
+    const history = createMemoryHistory();
+    const { getByText, getByLabelText } = render(
+      <Router history={history}>
```
history здесь не нужно

satansdeer commented on src/modules/logIn/LogIn.test.tsx:
```diff
@@ -0,0 +1,72 @@
+  });
+
+  describe('on submit', () => {
+    it('validates that email is  filled in', async () => {
```
лишний пробел

satansdeer commented on src/modules/logIn/LogIn.test.tsx:
```diff
@@ -0,0 +1,72 @@
+        fireEvent.click(getByText(/Войти/i));
+      });
+
+      expect(getByText('Пароль обязательное поле')).toBeInTheDocument();
```
проверь позитивный сценарий

satansdeer commented on src/modules/orderCheckout/Form.tsx:
```diff
@@ -0,0 +1,150 @@
+                name="cardMonth"
+                ref={register}
+              >
+                {cardMoths.map((month) => (
```
months

satansdeer reviewed (commented):
https://www.loom.com/share/6ff9012862dc4d6c86e02bb8d0b485c7

ilkovecVladislav committed:
refactor: change folder structure

Change folder structure. Removed modules folder and put the code form modules folder to pages
folder. Added checkbox component. Updated pizza constructor.
